scripts/load_meeting_prot.py

```diff
@@ -14,125 +17,192 @@
     sorted_name_dict = sorted(name_dict.iteritems(), key=operator.itemgetter(1))
     return sorted_name_dict
 
-def pars_file(file, ongoing_deliveries, content):
-    """parses output from script "update_checklist.py", which is loading runinfo
-    from the trello board. ongoing_deliverues is a dict of info from old meeting
-    that will be merged with the new info feched from the trello."""
-    coming_deliveries = {}
-    f=open(file,'r')
-    content=f.readlines()
-    nlines=len(content)
-    data = coming_deliveries
-    i=0
-    proj=None
-    while i < nlines:
-        row = content[i]
-        if row.strip() == "Ongoing":
-            data = ongoing_deliveries
-        if not row.strip()=='':
-            isproj=row.split()[0]+'XXX'
-            if isproj[1]=='.' or isproj[2]=='.':
-                proj = row.split()[0].strip()
-                if not proj in data.keys():
-                    data[proj]={'info':' '.join(row.split()[1:]), 'flowcells':{}}
-            elif proj and row.split()[0][0]!='1':
-                if data[proj].has_key('info'):
-                    if row.strip() not in data[proj]['info']:
-                        data[proj]['info'] = data[proj]['info']+' - '+row
-            elif proj and row.split()[0][0]=='1':
-                if not data[proj]['flowcells'].has_key(row.strip()):
-                    data[proj]['flowcells'][row.strip()]=[]
-                proj=None
-            i=i+1
-        else:
-            i=i+1
-    return coming_deliveries, ongoing_deliveries
-
-def update(sorted_names, col, info, ss_key, ws_key):
-    """Uppdates the new meeting protocol with old and new info."""
-    i=2
-    for name in sorted_names:
-        name = name[0]
-        CLIENT.UpdateCell(i, col, name , ss_key, ws_key)
-        CLIENT.UpdateCell(i, col+1, info[name]['info'] , ss_key, ws_key)
-        fcs = sorted(info[name]['flowcells'].keys())
-        for fc in fcs:
-            i=i+1
-            j=i
-            CLIENT.UpdateCell(i, col+1, fc, ss_key, ws_key)
-            for comment in info[name]['flowcells'][fc]:
-                CLIENT.UpdateCell(j, col+2, comment, ss_key, ws_key)
-                j=j+1
-            if i!=j:
-                i=j-1
-        i=i+2
-
-def main(old_wsheet, new_wsheet, file, ill_fix_it_myself):
-    if os.path.exists("tempo.txt"): os.system("rm tempo.txt")
-    if ill_fix_it_myself:
-        old_wsheet = date.today()-timedelta(days=7)
-        old_wsheet = old_wsheet.isoformat()
-        new_wsheet = date.today().isoformat()
-        file = "tempo.txt"
-        os.system("update_checklist.py hugin-conf.yaml >> tempo.txt")
-    ssheet_title='Genomics_Platform_Bioinformatics_Meeting_Deliveries'
-    ssheet = bcbio.google.spreadsheet.get_spreadsheet(CLIENT,ssheet_title)
+def starts_with_project_name(string):
+    "checks if string starts with project name"
+    if len(string) > 2:
+        if string[1] == '.' or string[2] == '.':
```

Comment:
@mayabrandi here you can consider to use a more advance check with regular expressions, something like 

```
is_project_name = re.compile('\w{1,2}\.\w+\_\d\d\_\d\d')
if is_project_name(string):

```
N.B. I am not 100% sure that this re works but something really similar should.